examples/neighbors/plot_lof.py

```diff
@@ -0,0 +1,51 @@
+"""
+=================================================
+Anomaly detection with Local Outlier Factor (LOF)
+=================================================
+
+This example uses the LocalOutlierFactor estimator
+for anomaly detection.
+"""
+print(__doc__)
+
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.neighbors import LocalOutlierFactor
+
+np.random.seed(42)
+
+# Generate train data
+X = 0.3 * np.random.randn(100, 2)
+X_train = np.r_[X + 2, X - 2]
+# Generate some regular novel observations
+X = 0.3 * np.random.randn(20, 2)
+X_test = np.r_[X + 2, X - 2]
+# Generate some abnormal novel observations
+X_outliers = np.random.uniform(low=-4, high=4, size=(20, 2))
+
+# fit the model
+clf = LocalOutlierFactor()
+clf.fit(X_train)
+y_pred_train = clf.predict(X_train)
+y_pred_test = clf.predict(X_test)
+y_pred_outliers = clf.predict(X_outliers)
+
+# plot the line, the samples, and the nearest vectors to the plane
```

Comment:
I don't think the comment corresponds to what is done. There is no line or plane in the plot.